.github/workflows/central-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish to the Ballerina central
+
+  on:
+    workflow_dispatch:
+      releaseVersion:
+        description: 'Release Version'
+        required: true
+
+  jobs:
+    publish-release:
+      runs-on: ubuntu-latest
+      if: github.repository_owner == 'ballerina-platform'
+      steps:
+        -   uses: actions/checkout@v2
+        -   name: Checkout Tag
+            run: git checkout v${{ github.event.inputs.ballerinaVersion }}
```

Comment:
Anyway, we have to create this branch manually right?

Reply:
No this will checkout the release tag

Reply:
Ah the command should be changed to 
git checkout v${{ github.event.inputs.releaseVersion }}